File: ProjetSymf/EvenementBundle/Entity/Event.php
<?php

namespace EvenementBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Form\Extension\Core\Type\DateType;
use Symfony\Component\Validator\Constraints as Assert ;

/**
 * Event
 *
 * @ORM\Table(name="Event")
 * @ORM\Entity(repositoryClass="EvenementBundle\Repository\EventRepository")
 */
class Event
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="Nom", type="string", length=255)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="Description", type="string", length=255)
     */
    private $description;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="DateDeb", type="date")
     */
    private $dateDeb;

    /**
     * @ORM\Column(name="date_fin",type="datetime")
     * @Assert\Expression(
     *    "this.getDateFin() > this.getDateDeb()",
     *    message="La date fin de l'evenement  doit etre supérieure à la date début"
     * )
     */
    private $dateFin;

    /**
     * @var string
     *
     * @ORM\Column(name="Lieu", type="string", length=255)
     */
    private $lieu;

    /**
     * @var \String
     *
     * @ORM\Column(name="nom_organisateur", type="string")
     */
    private $nomOrganisateur;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="HeureDeb", type="time")
     */
    private $heureDeb;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="HeureFin", type="time")
     */
    private $heureFin;

    /**
     * @ORM\Column(type="string")
     *
     * @Assert\NotBlank(message="Please, upload the product brochure as a PDF file.")
     * @Assert\Image()
     */
    private $image;
    /**
     * @var integer
     *
     * @ORM\Column(name="participate", type="integer",options={"default" = 0})
     */
    private $participate='0';
    /**
     * @ORM\Column(type="integer" , nullable=true)
     */
    private $nbrparticipants = '0' ;

    /**
     * @var integer
     *
     * @ORM\Column(name="idu", type="integer",options={"default" = 0})
     */
    private $idu;
    /**
     * @var string
     *
     * @ORM\Column(name="nomuser", type="string", length=255)
     */

    private $nomuser;

    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set nom
     *
     * @param string $nom
     *
     * @return Event
     */
    public function setNom($nom)
    {
        $this->nom = $nom;

        return $this;
    }

    /**
     * Get nom
     *
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Event
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set dateDeb
     *
     * @param \DateTime $dateDeb
     *
     * @return Event
     */
    public function setDateDeb($dateDeb)
    {
        $this->dateDeb = $dateDeb;

        return $this;
    }

    /**
     * Get dateDeb
     *
     * @return \DateTime
     */
    public function getDateDeb()
    {
        return $this->dateDeb;
    }

    /**
     * Set dateFin
     *
     * @param \DateTime $dateFin
     *
     * @return Event
     */
    public function setDateFin($dateFin)
    {
        $this->dateFin = $dateFin;

        return $this;
    }

    /**
     * Get dateFin
     *
     * @return \DateTime
     */
    public function getDateFin()
    {
        return $this->dateFin;
    }

    /**
     * Set lieu
     *
     * @param string $lieu
     *
     * @return Event
     */
    public function setLieu($lieu)
    {
        $this->lieu = $lieu;

        return $this;
    }

    /**
     * Get lieu
     *
     * @return string
     */
    public function getLieu()
    {
        return $this->lieu;
    }

    /**
     * Set nomOrganisateur
     *
     * @param string $nomOrganisateur
     *
     * @return Event
     */
    public function setNomOrganisateur($nomOrganisateur)
    {
        $this->nomOrganisateur = $nomOrganisateur;

        return $this;
    }

    /**
     * Get nomOrganisateur
     *
     * @return string
     */
    public function getNomOrganisateur()
    {
        return $this->nomOrganisateur;
    }

    /**
     * Set heureDeb
     *
     * @param \DateTime $heureDeb
     *
     * @return Event
     */
    public function setHeureDeb($heureDeb)
    {
        $this->heureDeb = $heureDeb;

        return $this;
    }

    /**
     * Get heureDeb
     *
     * @return \DateTime
     */
    public function getHeureDeb()
    {
        return $this->heureDeb;
    }

    /**
     * Set heureFin
     *
     * @param \DateTime $heureFin
     *
     * @return Event
     */
    public function setHeureFin($heureFin)
    {
        $this->heureFin = $heureFin;

        return $this;
    }

    /**
     * Get heureFin
     *
     * @return \DateTime
     */
    public function getHeureFin()
    {
        return $this->heureFin;
    }
    /**
     * Set image
     *
     * @param string $image
     *
     * @return Event
     */
    public function setImage($image)
    {
        $this->image = $image;

        return $this;
    }

    /**
     * Get image
     *
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }
    /**
     * Constructor
     */
    public function __construct()
    {
        $this->participants = new \Doctrine\Common\Collections\ArrayCollection();
        $this->dateNow = new \DateTime('now');


    }



    /**
     * @return mixed
     */
    public function getNbrparticipants()
    {
        return $this->nbrparticipants;
    }

    /**
     * @param mixed $nbrparticipants
     */
    public function setNbrparticipants($nbrparticipants)
    {
        $this->nbrparticipants = $nbrparticipants;
    }

    /**
     * Set idu
     *
     * @param integer $idu
     *
     * @return Event
     */
    public function setIdu($idu)
    {
        $this->idu = $idu;

        return $this;
    }

    /**
     * Get idu
     *
     * @return integer
     */
    public function getIdu()
    {
        return $this->idu;
    }

    /**
     * Set nomuser
     *
     * @param string $nomuser
     *
     * @return Event
     */
    public function setNomuser($nomuser)
    {
        $this->nomuser = $nomuser;

        return $this;
    }

    /**
     * Get nomuser
     *
     * @return string
     */
    public function getNomuser()
    {
        return $this->nomuser;
    }




    /**
     * Set participate
     *
     * @param integer $participate
     *
     * @return Event
     */
    public function setParticipate($participate)
    {
        $this->participate = $participate;

        return $this;
    }

    /**
     * Get participate
     *
     * @return integer
     */
    public function getParticipate()
    {
        return $this->participate;
    }
}

File: tesst/src/entities/event.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.util.Date;

/**
 *
 * @author Hp
 */
public class event {
    
     private static int id_courant ;

    public static int getId_courant() {
        return id_courant;
    }

    public static void setId_courant(int id_courant) {
        event.id_courant = id_courant;
    }
    
    private int id ; 
    private String nom;
    private String description;
   
//    private java.util.Date datedeb = new java.util.Date();
//    private java.util.Date datefin = new java.util.Date();
//    
    
    public java.util.Date dateDeb;
    private java.util.Date date_fin;
    private String lieu ;
    private String image ;
    private String nom_organisateur;
    private String lien ;
   


    private int idu; 
    private int nbrparticipants;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Date getDateDeb() {
        return dateDeb;
    }

    public void setDateDeb(Date dateDeb) {
        this.dateDeb = dateDeb;
    }

    public Date getDate_fin() {
        return date_fin;
    }

    public void setDate_fin(Date date_fin) {
        this.date_fin = date_fin;
    }

    public String getLieu() {
        return lieu;
    }

    public void setLieu(String lieu) {
        this.lieu = lieu;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getNom_organisateur() {
        return nom_organisateur;
    }

    public void setNom_organisateur(String nom_organisateur) {
        this.nom_organisateur = nom_organisateur;
    }

    public String getLien() {
        return lien;
    }

    public void setLien(String lien) {
        this.lien = lien;
    }

    public int getIdu() {
        return idu;
    }

    public void setIdu(int idu) {
        this.idu = idu;
    }

    public int getNbrparticipants() {
        return nbrparticipants;
    }

    public void setNbrparticipants(int nbrparticipants) {
        this.nbrparticipants = nbrparticipants;
    }

    public event(int id, String nom, String description, Date dateDeb, Date date_fin, String lieu, String image, String nom_organisateur, String lien, int idu, int nbrparticipants) {
        this.id = id;
        this.nom = nom;
        this.description = description;
        this.dateDeb = dateDeb;
        this.date_fin = date_fin;
        this.lieu = lieu;
        this.image = image;
        this.nom_organisateur = nom_organisateur;
        this.lien = lien;
        this.idu = idu;
        this.nbrparticipants = nbrparticipants;
    }

    public event() {
    }

    @Override
    public String toString() {
        return "event{" + "id=" + id + ", nom=" + nom + ", description=" + description + ", dateDeb=" + dateDeb + ", date_fin=" + date_fin + ", lieu=" + lieu + ", image=" + image + ", nom_organisateur=" + nom_organisateur + ", lien=" + lien + ", idu=" + idu + ", nbrparticipants=" + nbrparticipants + '}';
    }

   
   



  
}
